csharp-basic-topics/StringBuilderType/StringBuilderType/StringBuilderMethods.cs

```diff
@@ -0,0 +1,148 @@
+﻿namespace StringBuilderType;
+using System.Text;
+
+public class StringBuilderMethods
+{
+    public void Instantiate()
+    {
+        string value = "How to instantiate a StringBuilder";
+        int index = value.IndexOf("H") + 7;
+        int capacity = 30;
+        int maxCapacity = 100;
+
+        // Instantiate a StringBuilder 
+        var sb = new StringBuilder();
+
+        // Instantiate a StringBuilder and define a capacity
+        sb = new StringBuilder(capacity);
+
+        // Instantiate a StringBuilder and define a capacity and maximum capacity
+        sb = new StringBuilder(capacity, maxCapacity);
+
+        // Instantiate a StringBuilder from a string.
+        sb = new StringBuilder(value);
+
+        // Instantiate a StringBuilder from string  with a default capacity.  
+        sb = new StringBuilder(value, capacity);
+
+        // Instantiate a StringBuilder from substring and define a capacity.  
+        sb = new StringBuilder(value, index, value.Length - index, capacity);
+
+    }
+
+    public StringBuilder Append()
+    {
+        var sb = new StringBuilder("Welcome, ");
+        sb.Append("I hope you learnt something");
+        return sb;
+    }
+
+    public StringBuilder Append(string basetext, int length = 1000)
+    {
+        var sb = new StringBuilder(length);
+        // Append some text.
+        foreach (var item in basetext)
+        {
+            sb.Append(item);
+        }
+        return sb;
+    }
+
+    public StringBuilder AppendLine()
+    {
+        var stringBuilder = new StringBuilder("Welcome, I hope you learnt something.");
+        stringBuilder.AppendLine();
+        return stringBuilder;
+    }
+
+    public StringBuilder AppendLine(string basetext)
+    {
+        var sb = new StringBuilder();
+        sb.Append(basetext);
+        sb.AppendLine();
+        return sb;
+    }
+
+    public StringBuilder AppendFormat(string basetext, string format)
+    {
+        StringBuilder sb = new StringBuilder();
```

Comment:
Use var please. Check for other instances where the type is explicit in the entire solution. You're not being consistent with it.